warewulf.spec.in

```diff
@@ -82,6 +84,7 @@ make
 
 
 %install
+export NO_BRP_STALE_LINK_ERROR=yes
```

Comment:
Is this more specific to suse builds?

Reply:
Yes, the overlay has a broken symlink for the systemd service for wwclient. The linter doesn't like it.